x/liquidity/spec/02_state.md

```diff
@@ -1,105 +1,117 @@
 
-`PoolBatch` defines the batch(es) of a given liquidity pool that contains indexes of deposit / withdraw / swap messages. Index param increments by 1 if the pool id is same.
+The PoolBatch state stores information about the target liquidity pool, the batch index, the height at the beginning of the batch, the last index of the messages, and batch execution status.
+
+The PoolBatch state stores information about the liquidity pool batch in this structure:
 
 ```go
 type PoolBatch struct {
-	PoolId           uint64  // id of target liquidity pool
-	Index            uint64  // index of this batch
-	BeginHeight      uint64  // height where this batch is begun
-	DepositMsgIndex  uint64  // last index of DepositMsgStates
-	WithdrawMsgIndex uint64  // last index of WithdrawMsgStates
-	SwapMsgIndex     uint64  // last index of SwapMsgStates
-	Executed         bool    // true if executed, false if not executed yet
+    PoolId           uint64  // id of target liquidity pool
+    Index            uint64  // index of this batch
+    BeginHeight      uint64  // block height when batch is created
+    DepositMsgIndex  uint64  // last index of DepositMsgStates
+    WithdrawMsgIndex uint64  // last index of WithdrawMsgStates
+    SwapMsgIndex     uint64  // last index of SwapMsgStates
+    Executed         bool    // true if executed, false if not executed
 }
 ```
 
-## Batch Msg States
+## DepositMsgState for Batch Messages
 
-`DepositMsgState` defines the state of deposit message that contains state information as it is processed in the next batch(s)
+The `DepositMsgState` state defines the state of deposit message that contains state information as it is processed in the next batch or batches.
 
 ```go
 type DepositMsgState struct {
-	MsgHeight  uint64 // height where this message is appended to the batch
-	MsgIndex   uint64 // index of this deposit message in this liquidity pool
-	Executed   bool   // true if executed on this batch, false if not executed yet
-	Succeeded  bool   // true if executed successfully on this batch, false if failed
-	ToBeDelete bool   // true if ready to be deleted on kvstore, false if not ready to be deleted
-	Msg        MsgDepositWithinBatch
+    MsgHeight  uint64 // block height where this message is appended to the batch
+    MsgIndex   uint64 // index of this deposit message in this liquidity pool
+    Executed   bool   // true if executed on this batch, false if not executed
+    Succeeded  bool   // true if executed successfully on this batch, false if failed
+    ToBeDelete bool   // true if ready to be deleted on kvstore, false if not ready to be deleted
+    Msg        MsgDepositWithinBatch
 }
 ```
 
-`WithdrawMsgState` defines the state of withdraw message that contains state information as it is processed in the next batch(s)
+## WithdrawMsgState for Batch Messages
+
+The `WithdrawMsgState` state defines the state of the withdraw message that contains state information as it is processed in the next batch or batches.
```

Comment:
similar question as line 55, I'm not clear on what this particular state is

Reply:
See the comment above. If you still don't understand, let me know.